Lib/test/test_smtplib.py

```diff
@@ -558,6 +559,38 @@ def testSendMessageMultipleResentRaises(self):
             smtp.send_message(m)
         smtp.close()
 
+    @ patch('email.utils.formatdate')
+    def testSendMessageAddDateIfMissing(self, mocked_date_obj):
+        current_date = 'Thu, 1 Jan 1970 17:42:00 +0000'
+
+        mocked_date_obj.return_value = current_date
+        m = email.mime.text.MIMEText('A test message')
```

Comment:
could you use an explicit name and not just one letter for a variable ?

Reply:
Hi @matrixise, First of all, thanks for the review, really appreciate :).

I re-used what was used in previous tests around (see the test_smtplib file in general).  I would need clarification around if you want me to rename all the tests variables, or just this new tests variables, or do you prefer I just leave it like the rest?

Regards,
Eric Lafontaine

Reply:
As discussed on Slack, this will slide as it would imply not following the already existing syntax in the test file.